sample.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/python
+
```

Comment:
After uploading a second patch set to a new branch (sample_1) and force pushing the new patch set onto this branch (sample_latest), the original review and comments left through PR UI remained.